x-pack/plugin/downsample/qa/mixed-cluster/src/javaRestTest/resources/tsdb-bulk-request.txt

```diff
@@ -0,0 +1,14 @@
+{"index": {}}
```

Comment:
What are these files needed for? I don't see any actual java rest test classes.

Reply:
I've removed this in https://github.com/elastic/elasticsearch/pull/106878/commits/5d095aec680af68c029d1bca4cee651c62fa5e10